rules/__tests__/prefer_to_be_defined.test.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const RuleTester = require('eslint').RuleTester;
+const rules = require('../../').rules;
+
+const ruleTester = new RuleTester();
+
+ruleTester.run('prefer_to_be_defined', rules['prefer-to-be-defined'], {
+  valid: ['expect(true).toBeDefined();'],
```

Comment:
Can you add some more examples here? Mostly to be sure we avoid stuff like #6 and #20